IssueTracker/IssueTracker.FileSystem/Data/IData/IMetaData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Models.Request;
+using Models.Response;
+
+namespace IssueTracker.FileSystem.Data.IData;
+
+public interface IMetaData
+{
+    IEnumerable<MetaDataResponse> GetAll();
+    Task CreateOrUpdateAsync(MetaDataRequest entity);
+    Task<bool> DeleteAsync(string id, string group);
```

Comment:
what do 'id' and 'group' mean?

Reply:
the rowkey and partition key, but the API doesn't have to know that